src/lib/deepCopy.js

```diff
@@ -5,14 +5,39 @@
  * @return    {Mixed}
  */
 module.exports = function deepCopy(obj) {
+  var copy;
+
   if (obj === null || typeof obj !== 'object') {
     return obj;
   }
-  var src = obj instanceof Array ? [] : {};
-  for (var key in obj) {
-    if (Object.prototype.hasOwnProperty.call(obj, key)) {
```

Comment:
We need this line. At some point I ran across another external library that nuked `hasOwnProperty` so I had to call it directly off of the Object.prototype.